junitparser/junitparser.py

```diff
@@ -514,7 +514,7 @@ class TestSuite(Element):
 
     testcase = TestCase
 
-    def __init__(self, name=None):
+    def __init__(self, name: str =None):
```

Comment:
For this to allow for `None` as well, this would have to read

```suggestion
    def __init__(self, name: str | None = None):
```